app/logic/auth.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Auth
+  extend self
+
+  ADMIN_USERS = Rails.application.credentials.admin_users.split(',').index_with { |_email| true }
+
+  def admin?(user)
+    return false unless user
+
+    ADMIN_USERS[user.email].present?
+  end
```

Comment:
Since we don't have the needed list in the credentials file, at least for now it fails. So I think for making local testing more comfortable we can use some guards here. What do you reckon?

```suggestion
  ADMIN_USERS = Rails.application.credentials.admin_users&.split(',')&.index_with { |_email| true }

  def admin?(user)
    return false if user.blank? || ADMIN_USERS.blank?

    ADMIN_USERS[user.email].present?
  end
```

Reply:
We have encrypted credentials for test env as well. Don't worry

Reply:
I see, but it fails right now, so if we will add the list of emails or add guards then it would not be a problem.
can we do anything about it in this PR?